_posts/0000-01-02-Knicole80.md

```diff
@@ -0,0 +1,5 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Use the left arrow to go back!
```

Comment:
![This is a big pumpkin] (https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.pinterest.com%2Fpin%2F382594930823256574%2F&psig=AOvVaw1NgwukKT_IT3wnENSd12Ut&ust=1640753458935000&source=images&cd=vfe&ved=0CAsQjRxqFwoTCNjr0I_ZhfUCFQAAAAAdAAAAABAF)

Reply:
Thank you!